### PR TITLE
Upgrade OPAE to v1.4.0-1

### PR DESCRIPTION
**Pull request by bart0sh**

### CODE REVIEW

**Comment by bart0sh** [comment]
@hectoragb test failure seems to be caused by CI issue:

```
[Pipeline] withDockerRegistry

$ docker login -u k8s-bot -p ******** https://cloud-native-image-registry.westus.cloudapp.azure.com/

WARNING! Using --password via the CLI is insecure. Use --password-stdin.

Error response from daemon: Get https://cloud-native-image-registry.westus.cloudapp.azure.com/v2/: x509: certificate has expired or is not yet valid

[Pipeline] // withDockerRegistry

Error when executing success post condition:

hudson.AbortException: docker login failed

	at org.jenkinsci.plugins.docker.commons.impl.RegistryKeyMaterialFactory.materialize(RegistryKeyMaterialFactory.java:102)

	at org.jenkinsci.plugins.docker.workflow.AbstractEndpointStepExecution2.doStart(AbstractEndpointStepExecution2.java:53)

	at org.jenkinsci.plugins.workflow.steps.GeneralNonBlockingStepExecution.lambda$run$0(GeneralNonBlockingStepExecution.java:77)

	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)

	at java.util.concurrent.FutureTask.run(FutureTask.java:266)

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)

	at java.lang.Thread.run(Thread.java:748)
```

Please fix ASAP. Thanks.

**Comment by mythi** [comment]
> Please fix ASAP. Thanks.

It should be fixed. Did you re-push today?

**Comment by bart0sh** [comment]
@mythi  no, I didn't know it's fixed. will do. Thanks for pointing this out to me.

File: demo/opae-nlb-demo/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ ARG CLEAR_LINUX_VERSION=
 RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
     swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc doxygen Sphinx patch
 # Fetch dependencies and source code
-ARG OPAE_RElEASE=1.3.2-1
+ARG OPAE_RElEASE=1.4.0-1
```
Comment (rojkov) [inline demo/opae-nlb-demo/Dockerfile]: nit: could you please rename it to `OPAE_RELEASE`?

Comment (bart0sh) [reply demo/opae-nlb-demo/Dockerfile]: done

**Pull request merged**